app/helpers/dams_objects_helper.rb

```diff
@@ -612,94 +525,6 @@ def grabIcon(fileUse)
 		return icon
 	end
 
-	#---
-	# Renders a node of the COV component tree.
-	#
-	# @param index The object's component index.
-	# @return nil
-	# @author David T.
-	#---
-	def displayNode(index)
```

Comment:
just a quick clarification question. what is replacing these 3 deleted functions? `displayNode, displayComponentTree, initComponentTree`? 

I see the test suite passed ( 👍 ) but I'm wondering where the code for these ended up.